benchmark/single-source/MapReduce.swift

```diff
@@ -133,7 +133,7 @@ public func run_MapReduceShortString(_ N: Int) {
 
 @inline(never)
 public func run_MapReduceClass(_ N: Int) {
-  let numbers = (0..<1000).map { NSDecimalNumber(value: $0) }
+  let numbers = (0..<1000).map { NSDecimalNumber(integerLiteral: $0) }
```

Comment:
I think it's preferred not to call the ExpressibleBy methods directly. Maybe it makes sense to replace this use of `NSDecimalNumber` with something else, given we now have `DecimalNumber`.